extensions/wikia/MercuryApi/tests/MercuryApiModelTest.php

```diff
@@ -12,19 +12,21 @@ public function setUp() {
 	 */
 	private function setupGlobals() {
 		$this->mockGlobalVariable('wgLoadAdsInHead', true);
+		$this->mockGlobalVariable('wgAdDriverBottomLeaderboardImpressionCapping', false);
```

Comment:
Do we want to check what happens if the variable is not set in WikiFactory? Do we want to test how it behaves with `true` and some falsy values like `""`?